packages/cli/src/sso/saml/routes/saml.controller.protected.ee.ts

```diff
@@ -32,11 +33,12 @@ samlControllerProtected.post(
 	SamlUrls.config,
 	samlLicensedOwnerMiddleware,
 	async (req: SamlConfiguration.Update, res: express.Response) => {
-		const result = await SamlService.getInstance().setSamlPreferences({
-			metadata: req.body.metadata,
-			mapping: req.body.mapping,
-		});
-		return res.send(result);
+		if (isSamlPreferences(req.body)) {
```

Comment:
What about if we define a schema for the config like in the LDAP feature (see [here](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/Ldap/helpers.ts#L84))? that way we can show an error describing exactly what is wrong in the body.